worksheets/rollups/comparisons.http

```diff
@@ -0,0 +1,198 @@
+###
+
+#21, 14, 3, 11, 24, 1, 1, 41, 14, 13
```

Comment:
What is this used for?

Reply:
A secret code

I kid I kid, probably some numbers I was keeping track of for comparing raw rollup data vs rollup data in below queries.

Reply:
are you removing these Drew?

Reply:
I wasn't planning on it as it's used for comparison in the query below, but I can if someone feels strongly about it.
Again though I wouldn't look *too* much into this PR. These are just sample queries added to be used when running a local cluster for some quick sanity checks among other things.